test.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_s3_bucket" "acme-s3-access-logging" {
+  bucket = "test"
+}
+
+resource "aws_s3_bucket" "not-another-public-one-please" {
+  bucket = "my-public-bucket"
+  acl    = "public-read"
```

Comment:
## result
tfsec check aws-s3-no-public-access-with-acl failed.
## severity
⚠️CRITICAL
## reason
Resource 'aws_s3_bucket.not-another-public-one-please' has an ACL which allows public access.
## how to ignore
`#tfsec:ignore:AWS001`([refs](https://github.com/aquasecurity/tfsec#ignoring-warnings))

For more information, [see](https://tfsec.dev/docs/aws/s3/no-public-access-with-acl#aws/s3)